crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -19,6 +19,7 @@
 require "json"
 
 class BarclampController < ApplicationController
+  skip_before_filter :enforce_installer, if: proc { Crowbar::Installer.steps_done.include?(:initial_chef_client) }
```

Comment:
Metrics/LineLength: Line is too long. [114/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)